From 68d30adde982f1be33a934707fa105c0db6f7f8f Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Tue, 17 Jan 2012 20:59:56 +0000
Subject: [PATCH 4/8] Update use of error_connection under WIN32 to
 _xcb_conn_ret_error()

Unfortunately, commit 31b57676 adding WSACleanup/WSAShutdown on Win32 adds a new use
of error_connection, which was removed in commit 769acff0, applied 5 minutes earlier.

src/xcb_util.c: In function 'xcb_connect_to_display_with_auth_info':
src/xcb_util.c:433:39: error: 'error_connection' undeclared (first use in this function)

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
Reviewed-by: Arvind Umrao <arvind.umrao@oracle.com>
Signed-off-by: Uli Schlachter <psychon@znc.in>
---
 src/xcb_util.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git libxcb-1.8/src/xcb_util.c libxcb-1.8/src/xcb_util.c
index 9329b81..7173db3 100644
--- libxcb-1.8/src/xcb_util.c
+++ libxcb-1.8/src/xcb_util.c
@@ -430,7 +430,7 @@ xcb_connection_t *xcb_connect_to_display_with_auth_info(const char *displayname,
 #ifdef _WIN32
         WSADATA wsaData;
         if (WSAStartup(MAKEWORD(2, 2), &wsaData) != 0) {
-            c = (xcb_connection_t *) &error_connection;
+            c = _xcb_conn_ret_error(XCB_CONN_ERROR);
             goto out;
         }
 #endif
-- 
1.7.9.2

